system-int-test/test/cmr/system_int_test/search/humanizer/community_usage_metrics_test.clj

```diff
@@ -61,7 +61,7 @@
 
   (testing "Create with unknown token"
     (is (= {:status 401
-            :errors ["Token [ABC] does not exist"]}
+            :errors ["You do not have permission to perform that action."]}
```

Comment:
The old error was more informative. Any chance we could make the error message `"Token [ABC] does not exist or has expired"`?

Reply:
I'm thinking this is because there is some indirect token validation happening somewhere. Looking.

Reply:
so it looks like its hidden in context-augmenter/add-user-id-and-sids-handler which is a part of the routes handler.  I missed this the first time around.